tests/testthat/test-submit-model.R

```diff
@@ -61,6 +61,7 @@ test_that("submit_model(.dry_run=T) with bbi_nonmem_model object parses correctl
 test_that("submit_model() creates correct call for non-NULL .config_path", {
   temp_config <- tempfile(fileext = ".yaml")
   readr::write_file("foo", temp_config)
```

Comment:
shouldn't the `normalizePath` be _above_ the `write_file` call? Is there a reason you don't just do 
```
temp_config <- normalizePath(tempfile(fileext = ".yaml"))
```
?

Reply:
The call to `normalizePath()` must happen after `write_file()`, because the path given by `temp_config()` does not exist until after that. `tempfile()` does not _create_ a file, but rather returns

> a character vector giving the names of possible (temporary) files. Note that no files are generated by tempfile.

Thus, calling `normalizePath(tempfile())` will raise a warning, because the temporary file does not yet exist. But, after we write to it (or otherwise create it), it will exist, and `normalizePath()` will work as intended.

```r
x <- tempfile()
x
#> [1] "/var/folders/d3/x8vvcfl93bzdqyjldg5kj6cr0000gp/T//RtmplPceD2/file591b28bc8217"
normalizePath(x)
#> Warning in normalizePath(x): path[1]="/var/folders/d3/
#> x8vvcfl93bzdqyjldg5kj6cr0000gp/T//RtmplPceD2/file591b28bc8217": No such file or
#> directory
#> [1] "/var/folders/d3/x8vvcfl93bzdqyjldg5kj6cr0000gp/T//RtmplPceD2/file591b28bc8217"
file.exists(x)
#> [1] FALSE
writeLines("foo", x)
normalizePath(x)
#> [1] "/private/var/folders/d3/x8vvcfl93bzdqyjldg5kj6cr0000gp/T/RtmplPceD2/file591b28bc8217"
```